src/platform/Ameba/ConfigurationManagerImpl.cpp

```diff
@@ -152,6 +152,30 @@ CHIP_ERROR ConfigurationManagerImpl::GetPrimaryWiFiMACAddress(uint8_t * buf)
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR ConfigurationManagerImpl::GetUniqueId(char * buf, size_t bufSize)
+{
+    ReturnErrorCodeIf(bufSize != 32, CHIP_ERROR_INVALID_MESSAGE_LENGTH);
```

Comment:
the CHIP_DEVICE_CONFIG_UNIQUE_ID below has its own size check. Should this check not be on the `#else` branch?

Reply:
This has been modified, thanks!